source/blog/2020-10-27-this-month-in-solidus-september-2020.html.markdown

```diff
@@ -0,0 +1,43 @@
+- [We now only allow capturing or voiding payments with a positive amount.](https://github.com/solidusio/solidus/pull/3761) Capturing or voiding negative payments doesn't make too much sense. Thanks Andrea Longhi for this update!
+- [Fixed a bug where the default tax category was not being displayed on the product page.](https://github.com/solidusio/solidus/pull/3759) Even when a default tax category was picked, the product page still showed "none" as the selected option. This PR fixes that - thanks Valentín Sanjuan! Also, this was Valentíns first PR submission to Solidus. We're always happy to see new contributors!
+- [Exposed a refresh() method for sortable tables in JS.](https://github.com/solidusio/solidus/pull/3754) This allows us to call `refresh()` on a sortable table on demand. Thanks to Github user `brunoao86` for this change - and for being another first-time contributor to Solidus. Welcome to the club!
+- [Added tests for deprecation warnings.](https://github.com/solidusio/solidus/pull/3753) This PR adds some tests for expected deprecation warnings, instead of just ignoring them in our test suite. Thanks to Dumitru Ceban for this change!
+- [Fixed incorrect resource error handling.](https://github.com/solidusio/solidus/pull/3728) On a nil resource, Solidus should throw a 404 error and redirect to the resource index page, however in some cases a 500 error was thrown directing to an error page. This PR fixes that issue. Thanks to first-time contributor Ikraam Ghoor for this change!
+- [Removed custom CanCanCan action aliases.](https://github.com/solidusio/solidus/pull/3701) This removes and deprecates usages of CanCanCan custom action aliases, as they're not needed in newer versions of CanCanCan and can (can can) make upgrading to new versions harder. Thanks to Filippo Liverani for this change.
+- [Removed an N+1 query on the shipment model.](https://github.com/solidusio/solidus/pull/3598) This simply adds `products` to an includes statement, which just so happens to drastically decrease the amount of DB queries needed. Thanks to first-time contributor Alban for this crucial change!
+
+### Solidus 2.11 IS OUT NOW!
+
+*BIG NEWS*! Solidus 2.11 is now out and ready to be used! This update will be the last until Solidus 3.0. It's important to note that the only change we'll be making in 3.0 is to remove all deprecated functionality - so be on the lookout for deprecation warnings in this release, as you'll need to fix those before updating to 3.0!
+
+Be on the lookout for more information about Solidus 3.0 - we're excited to be approaching our next major version release!
```

Comment:
There's a repetition: you use "be on the lookout" in the previous paragraph as well. Perhaps "Stay tuned" could work here?